### PR TITLE
fix: Normalize error messages

### LINKED ISSUE #7026: Error message is typed twice when invalid token is tried to input in the settings
Your issue may already be reported! Please search on the [issue tracker](https://github.com/keptn/keptn/issues) before creating a new one.

**Environment**
Keptn version: 0.12.2
Bridge version: 0.12.2

* Client OS (e.g., Linux, macOS, Windows):
* Keptn Version (`keptn version`):
* Kubernetes Cloud Provider (e.g., GKE, AKS):
* Kubernetes version (`kubectl version`):

**Affected Component**

* [ ] Approval
* [ ] Bridge
* [ ] CLI
* [ ] Datastore
* [ ] Docs
* [ ] Distributor / Event Broker
* [ ] Helm
* [ ] Istio
* [ ] Jmeter
* [ ] Kubernetes Integration
* [ ] Openshift Integration
* [ ] REST API
* [ ] Webhook

**Describe the bug**

**To Reproduce**
It's quite simple to reproduce the issue. You have to have a project setup with the git upstream. Just go to the Project settings of the page and try to input any invalid token for the upstream.
You will see the error message is printed twice in the page like showed in the image below:
image

**Expected behavior**

**Current behavior**

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by odubajDT**
## This PR

- normalizes error messages

### Related Issues

Fixes #7026 

### Integration tests
https://github.com/keptn/keptn/actions/runs/1951503358

### CODE REVIEW

File: shipyard-controller/handler/loghandler.go [hunk 1]
```diff
@@ -65,13 +67,13 @@ func (lh *LogHandler) CreateLogEntries(context *gin.Context) {
...
 	if err := context.ShouldBindQuery(params); err != nil {
-		SetBadRequestErrorResponse(err, context, "Invalid request format")
+		SetBadRequestErrorResponse(context, fmt.Sprintf(InvalidRequestFormatMsg, err.Error()))
 		return
 	}
 
 	logs, err := lh.logManager.GetLogEntries(*params)
 	if err != nil {
-		SetInternalServerErrorResponse(err, context, "Unable to retrieve logs")
+		SetInternalServerErrorResponse(context, fmt.Sprintf(UnableRetrieveLogsMsg, err.Error()))
```
Comment (thisthat) [inline shipyard-controller/handler/loghandler.go:76]: Should this be 404? 🤔

File: shipyard-controller/handler/error.go [hunk 2]
```diff
@@ -29,3 +29,23 @@ var ErrNoMatchingEvent = errors.New("no matching event found")
...
+
+var InvalidPayloadMsg = "Could not validate payload: %s"
+
+var NoProjectNameMsg = "Must provide a project name"
+
+var NoServiceNameMsg = "Must provide a service name"
+
+var UnableQueryStateMsg = "Unable to query sequence state repository: %s"
+
+var UnableControleSequenceMsg = "Unable to controle sequence: %s"
```
Comment (warber) [inline shipyard-controller/handler/error.go:47]: controle --> control type, also in other places

File: secret-service/pkg/handler/common.go [hunk 3]
```diff
@@ -1,58 +1,43 @@
...
+
 	"github.com/gin-gonic/gin"
 	"github.com/keptn/keptn/secret-service/pkg/model"
-	"net/http"
-	"strings"
 )
 
-func SetBadRequestErrorResponse(err error, c *gin.Context, message ...string) {
-	msg := errMsg(err, message)
+var ErrCreateSecret = "Unable to create secret: %s"
```
Comment (warber) [inline secret-service/pkg/handler/common.go:10]: pls rename these to `Err...Msg`

File: shipyard-controller/handler/projecthandler.go [hunk 4]
```diff
@@ -100,11 +101,11 @@ func (ph *ProjectHandler) GetProjectByName(c *gin.Context) {
 	project, err := ph.ProjectManager.GetByName(projectName)
 	if err != nil {
 		if project == nil && err == ErrProjectNotFound {
```
Comment (thisthat) [inline shipyard-controller/handler/projecthandler.go:103]: Should we change also this line to use `errors.Is()`? Like here: https://github.com/keptn/keptn/pull/7080/files#diff-6d643b14d1fbb5301d0aed6dd99b501b9e249daa4dceb265749974aeb853dd24R201

**Review by warber — CHANGES_REQUESTED** [review]

File: shipyard-controller/handler/loghandler.go (hunk 1 above)
Comment (odubajDT) [reply shipyard-controller/handler/loghandler.go:76]: We can change it

File: shipyard-controller/handler/projecthandler.go (hunk 4 above)
Comment (odubajDT) [reply shipyard-controller/handler/projecthandler.go:103]: sure

File: secret-service/pkg/handler/common.go (hunk 3 above)
Comment (odubajDT) [reply secret-service/pkg/handler/common.go:10]: ok

**Review by warber — APPROVED** [review]

**Pull request merged**